### PR TITLE
Docstring and fix generate grountruth

### PR DESCRIPTION
**Pull request by Kh4L**
Fix docstring

### CODE REVIEW

File: python/raft-ann-bench/src/raft_ann_bench/generate_groundtruth/__main__.py [hunk 1]
```diff
@@ -96,18 +96,18 @@ def main():
         "The input and output files are in big-ann-benchmark's binary format.",
         epilog="""Example usage
     # With existing query file
-    python -m raft_ann_bench.generate_groundtruth --dataset /dataset/base.\
-fbin --output=groundtruth_dir --queries=/dataset/query.public.10K.fbin
+    python -m raft_ann_bench.generate_groundtruth --output=groundtruth_dir \
+--queries=/dataset/query.public.10K.fbin /dataset/base.fbin
```
Comment (divyegala) [inline python/raft-ann-bench/src/raft_ann_bench/generate_groundtruth/__main__.py:100]: What has changed here? I think we would like to keep the --dataset argument

**Review by tfeher — CHANGES_REQUESTED** [review]
Thanks @Kh4L for this PR! In the PR description you write that "sqeuclidean is not supported in raft-ann-bench". I think that is not correct, I have tested in 24.06 and 24.08, and it is supported in both cases. Therefore I would recommend to keep the default metric "sqeuclidean". The rest of the docstring change look good to me.

**Review by tfeher — APPROVED** [review]
Thanks Serge, LGTM!

**Comment by cjnolet** [comment]
Benchmarking tool has been migrated over to cuVS. Closing

**Pull request closed without merging**